hs_core/templates/pages/genericresource.html

```diff
@@ -830,14 +830,13 @@ <h4 class="modal-title">Manage access</h4>
 
         {#        ======= Content =======#}
 
+        {% if cm.resource_type != "ToolResource" %}
```

Comment:
Those are actually two different use cases. show_content_files is used to decide whether a) to display the content files or b) to show a flag to the user indicating no access to the files. 

For this case we want to hide the content section because it is not applicable to this type of resource.